react/training/src/redux/login/actions.js

```diff
@@ -0,0 +1,25 @@
+import { getByEmail } from '../../services/loginService';
+
+export const actionTypes = {
+  LOGIN_SUCCESS: 'LOGIN_SUCCESS',
+  LOGIN_FAILURE: 'LOGIN_FAILURE',
+  LOGIN_REQUEST: 'LOGIN_REQUEST'
+};
+
+export const loginActions = {
+  handleSubmit: userLogged => async dispatch => {
+    dispatch({ type: actionTypes.LOGIN_REQUEST, payload: true });
```

Comment:
You don't need to pasas `payload: true`. Just change your `isLoading: true` directly in the store. Payload is usually used to pass data from API or another part.